bench/utils.py

```diff
@@ -617,6 +617,26 @@ def update_node_packages(bench_path='.'):
 		update_yarn_packages(bench_path)
 
 
+def install_python_dev_dependencies(bench_path='.'):
+	from bench.app import get_apps
+
+	print("Installing development dependencies")
+
+	env_py = _get_venv_python(bench_path)
+	for app in get_apps():
+		app_path = os.path.join(bench_path, "apps", app)
+		dev_requirements_path = os.path.join(app_path, "dev-requirements.txt")
+
+		if os.path.exists(dev_requirements_path):
+			print(f'\n{color.yellow}Installing development dependencies for {app}{color.nc}')
+			exec_cmd(f"{env_py} -m pip install -q -r {dev_requirements_path}", cwd=bench_path)
+		else:
+			print(f'\n{color.red} dev-requirements.txt not found in {app}{color.nc}')
```

Comment:
Use `bench.utils.log` instead. Common interface to log bench commands etc and add colour.